rust-client/src/main.rs

```diff
@@ -51,8 +48,16 @@ pub enum Command {
 
 #[derive(Debug, Parser)]
 pub enum UserCommand {
-    Deposit { token_amount: u64 },
-    Withdraw { unmint_amount: u64 },
+    Deposit {
+        token_amount: u64,
+    },
+    Withdraw {
+        unmint_amount: u64,
+    },
+    WithdrawFromStrategy {
+        unmint_amount: u64,
+        strategy: Pubkey,
```

Comment:
Should we put a mapping of protocol to strategy pubkey at the README ?

Reply:
User can read strategy pubkey through vault state